src/components/PropertyCreate/PropertyInfo.js

```diff
@@ -1326,13 +1586,17 @@ function PropertyInfo() {
                             style={{
                               backgroundColor: "red",
                             }}
-                            value="1 Bike"
+                            value="1"
                           >
                             1 Bike
                           </option>
-                          <option value="2 Bikes">2 Bikes</option>
-                          <option value="Included with Car">Included with Car</option>
-                          <option value="">No Bike Parking</option>
+                          <option value="2">2 Bikes</option>
```

Comment:
why are we just not changing the enum itself?
why are we doing this roundabout mapping? @shlsystemica

Reply:
It was already there, and all these validation is supplies by api only, maybe i can not define enum itself